src/main/java/com/parsing/exception/PDFParsingException.java

```diff
@@ -0,0 +1,12 @@
+package com.parsing.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+public class PDFParsingException extends RuntimeException{
```

Comment:
looks like there is no auto refactor

Reply:
Done.

Reply:
but it should be a space before "{" not sure if auto-format does it.

Reply:
ok